modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -281,6 +295,35 @@ typedef struct indigo_core_gentable_ops {
         void *table_priv, void *entry_priv, of_list_bsn_tlv_t *key,
         of_desc_str_t err_txt);
 
+    /**
+     * @brief Add an entry
+     * @param params paramters 
+     */
+    indigo_error_t (*add4)(
+        indigo_cxn_id_t cxn_id,
+        void *table_priv, of_list_bsn_tlv_t *key, of_list_bsn_tlv_t *value,
+        void **entry_priv, of_desc_str_t *err_txt,
+        of_object_t *obj);
+
+    /**
+     * @brief Modify an entry
+     */
+    indigo_error_t (*modify4)(
+        indigo_cxn_id_t cxn_id,
+        void *table_priv, void *entry_priv, of_list_bsn_tlv_t *key,
+        of_list_bsn_tlv_t *value, of_desc_str_t *err_txt,
+        of_object_t *obj);
+
+    /**
+     * @brief Delete an entry
+     */
```

Comment:
so what's the use case for the "force" parameter?
if the lower level driver code isn't going to call the resume function, then why wouldn't it just return INDIGO_ERROR_NONE so that the state manager can just clean up and move on?
i'm thinking that the "force" parameter is not necessary.